front/lib/api/data_source_view.ts

```diff
@@ -285,3 +287,48 @@ export async function getContentNodesForDataSourceView(
     total: contentNodesResult.total,
   });
 }
+
+export class NotAuthorizedError extends Error {}
```

Comment:
Can we avoid creating one error class per type? We have an existing class `DustError` that partially covers those needs.

Reply:
Sure !